.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# Default code owners
+*                           @openebs/mayastor-approvers @openebs/ci-buildscript-approvers
+
+# OEP owners
+/designs                    @openebs/org-maintainers
```

Comment:
There's no designs and no security here, also not needed, this is internal CI repo?

Reply:
Makes sense.